ipam/claim.go

```diff
@@ -51,7 +51,15 @@ func (c *claim) Try(alloc *Allocator) bool {
 	if !alloc.ring.Contains(c.cidr.Addr) {
 		// Address not within our universe; assume user knows what they are doing
 		alloc.infof("Address %s claimed by %s - not in our range", c.cidr, c.ident)
-		addOwned()
+		previousOwner := alloc.findOwner(c.cidr.Addr)
+		switch {
+		case previousOwner == "":
+			addOwned()
+		case previousOwner == c.ident: // already owned by this ID
+		case c.ident == api.NoContainerID: // already owned by anonymous container
+		default:
```

Comment:
We have just done a `docker stop c2`, so what is happening is IPAM is waiting 30 seconds to see if we restart that container.

We should probably allow a claim from a dead container. I'll try changing the code.